### ISSUE TITLE
Language Selection not accessible for Screenreader

### ISSUE THREAD

**Issue by Distaly** [LAYAIraq/LAYA-Iraq-Frontend#288]
The language selection only reads the abbreviation for languages. Either need to expand the name in the icons object, or dynamical bind a aria-label.
